addon/components/radio-button-input.js

```diff
@@ -50,7 +54,7 @@ export default Component.extend({
 
     if (groupValue !== value) {
       this.set('groupValue', value); // violates DDAU
-      run.once(this, 'sendChangedAction');
+      run.once(this, 'sendChangedAction', value);
```

Comment:
I think reading the value when `sendChangedAction` runs is preferable because if `value` changes between when this line executes and when `sendChangedAction` this approach would potentially have us only send an old value